lib/Doctrine/Migrations/Metadata/ExecutedMigration.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Metadata;
+
+use DateTime;
+use Doctrine\Migrations\Version\Version;
+
+class ExecutedMigration
+{
+    /** @var Version */
+    private $version;
+
+    /** @var DateTime */
+    private $executedOn;
+
+    /**
+     * Milliseconds
+     *
+     * @var int
```

Comment:
Int|null until you add a value object ☺️

Reply:
Good idea. I was hoping that somehow doctrine Cs could do it automatically